pkg/agent/agent_windows.go

```diff
@@ -32,3 +40,127 @@ func (i *Initializer) setupExternalConnectivity() error {
 	}
 	return nil
 }
+
+// prepareHostNetwork creates HNS Network for containers.
+func (i *Initializer) prepareHostNetwork() error {
+	uplinkNetConfig := new(config.AdapterNetConfig)
```

Comment:
It's because I want to use `uplinkNetConfig ` instead of `i.nodeConfig.UplinkNetConfig` in the following lines.

`i.nodeConfig.UplinkNetConfig` is too long.... can I use in this way?

Reply:
You can use in this way. I just feel it is not necessary, but no strong preferrence.

Reply:
Done